### ISSUE TITLE
Bug: Links in navbar not working

### ISSUE THREAD

**Issue by Achal-00** [getAlby/bitcoin-connect#149]
The links in the navigation bar is not working.

https://github.com/getAlby/bitcoin-connect/assets/106076516/5efc2189-802d-4ae9-9f19-f3ba3325f773

**Comment by rolznz** [comment]
Thanks for reporting this, @Achal-00 !

It's only on the initial navbar, but when you scroll down it's fine.

**Comment by stackingsaunter** [comment]
It seems to be overrides related, I think when you added the payment modal perhaps? Because they should work and they did work on previous versions. Don't know how to fix it, so I just hid it for now @rolznz

**Comment by rolznz** [comment]
@stackingsaunter I don't believe it is related to overrides, they only apply to the connect wallet buttons, and there is no bitcoin connect button in the header.
